metrics/aocih.sh

```diff
@@ -31,12 +31,18 @@ age_in_hours=0
 cd "$(dirname "${java_file}")"
 
 if git status > /dev/null 2>&1; then
-    repo_first_commit=$(git log --reverse --format=%at | head -1)
-    file_first_commit=$(git log --diff-filter=A --format=%at -- "$java_file" | tail -1)
-    if [[ -n "$file_first_commit" ]]; then
+    repo_first_commit=$(git log --reverse --format=%at | head -n 1 || true)
+    file_first_commit=$(git log --diff-filter=A --format=%at -- "$java_file" | tail -n 1 || true)
+
```

Comment:
@dzhovi try to refrain from empty lines inside code blocks: https://www.yegor256.com/2014/11/03/empty-line-code-smell.html